src/pages/hero/Hero.jsx

```diff
@@ -0,0 +1,204 @@
+                <br /> to demonstrate
+              </div>
+              <LinkToID to="head" offset={-50}>
+                <button className="btn btn-cta">shop now</button>
+              </LinkToID>
+            </div>
+          </div>
+          <div className="banner img_curve">
+            <img className="" src="/assets/curve1.webp" alt="curve" />
+          </div>
+          <div className="text-center kodchasan hero_title text_cl">
+            <div className="h4">High-Quality VR </div>
+            <h2 className="h2">
+              <b>The Real World Immersion</b>
+            </h2>
+          </div>
+          {offerData.map((offer) => (
+            <div key={offer} id="head">
+              <div className="flex wrapper_head montserrat">
+                <h2 className="head_text">{offer}</h2>
+                <Link
+                  to={`/products/${offer
+                    .replace(/ /g, "")
+                    .replace(/\d+% ?/g, "")
+                    .toLowerCase()}`}
+                >
+                  <button className="btn-link view_all p">view all</button>
+                </Link>
+              </div>
+              <div className="wrapper_card flex">
+                {productData
+                  .filter((item) => item.offer.toString() === offer)
+                  .slice(0, 5)
+                  .map(
+                    ({
+                      _id,
+                      name,
+                      image,
+                      price,
+                      productName,
+                      ratings,
+                      inStock,
+                      material,
+                      fastDelivery,
+                      offer,
+                    }) => {
+                      return (
+                        <div key={_id}>
+                          <Link to={`/product/${_id}`}>
+                            {" "}
+                            <div className="card_hr flex-center card_hero">
+                              <div className="card_hr_img card_img_hero">
+                                <img src={image} alt={productName} />
+                              </div>
+                              <div className="card_hr_content">
+                                <p title={name} className="h3 card_name_hero">
+                                  {name}
+                                </p>
+                                <StarRatings
+                                  rating={ratings}
+                                  starDimension="2rem"
+                                  starSpacing="0px"
+                                  starRatedColor="orange"
+                                />
+                                <div className="card_hr_price text_cl">
+                                  <p className="p-lg">
+                                    deal of the day:
+                                    <span className="h2 dod_price">
+                                      {` $${parseFloat(price)}`}
```

Comment:
Not sure what's the use of double template string  '$' over here

Reply:
One is for currency '$' and other for template string 😅